tests/annotations1/a.scala

```diff
@@ -0,0 +1,2 @@
+class Annot1(s: String) extends scala.annotation.StaticAnnotation
```

Comment:
Are these tests meant to be under `tests/pos/` rather than directly under `tests/`?

Reply:
oh indeed

Reply:
Oops, updated